drainer/pump.go

```diff
@@ -134,6 +138,16 @@ func (p *Pump) PullBinlog(pctx context.Context, last int64) chan MergeItem {
 				continue
 			}
 
+			if isBinlogPurged {
+				// some binlogs have been purged in pump, just print the log and wait to exit by user.
+				pLog.Print(labelBinlogGCed, func() {
+					p.logger.Error("some binlogs have been purged in pump")
+				})
+
+				time.Sleep(time.Second)
```

Comment:
why not exit directly? since `isBinlogPurged` never set back to false.

Reply:
if exit directly, then systemd will restart it again (but that's meaningless). if not exist, log and metrics and still keep working.

for `isBinlogPurged`, the user should handle it manually and re-establish the replication.